### ISSUE TITLE
Bug: Holochain seems to have changed the behavior of paths.

### ISSUE THREAD

**Issue by adaburrows** [lightningrodlabs/rea-playspace#210]
This manifests itself in a few weird ways:

After creating a plan and closing the Launcher down and reopening it, you can no longer load the same plan again. You can create a new one, and then you can see there is a blank resource and blank process at the top of each list. In fact you can see a blank agent, resource specification, process specification, plan, position, etc. Everything path that used to be simply `"x"` now has a `"x".""` that needs to be filtered out or used to store the data associated with the object.

**Comment by fosterlynn** [comment]
I'm sure this is the same thing, but just to report that you don't have to close down the launcher for this to happen.   Just close the app, and re-open from the launcher.  Looks like if you don't close the app, you will still have access to it.

**Comment by adaburrows** [comment]
Yeah, this is a result of caching everything in memory locally. When the window running it closes it can't retrieve anything because it only chooses the first object for each path, which just so happens to be blank in the new version.
